Misc/NEWS.d/2.7.18.11.rst

```diff
@@ -30,3 +30,21 @@ Python2 doesn't support PAX headers so, for the most part this doesn't affect Py
 Various tests were added from the CVE fix to improve rigour
 
 [3.12] gh-121285: Remove backtracking when parsing tarfile headers (GH-121286) (GH-123543)
+
+.. bpo: ?
+.. date: 2025-01-20
+.. nonce: 
+.. release date: 2025-01-22
+.. section: xml_etree
+
+BE-4504 Use newer Expat > 2.6.3 (in particular AS Platform Expat 2.6.4)
+
+Expat is now stricter, and invalid IRIs are now rejected with a syntax error.
+
+    # This IRI being used (xmlns:prefix) isn't a valid IRI, and this test will never work, 
```

Comment:
This doesn't need to be in the news

Reply:
Okay, removed.

Reply:
I meant the copy/paste commented out test. But this whole change might not rise to news level